src/coreclr/gc/gc.cpp

```diff
@@ -2225,6 +2333,8 @@ size_t      gc_heap::g_promoted;
 size_t      gc_heap::g_bpromoted;
 #endif //BACKGROUND_GC
 
+const int n_heaps = 1;
```

Comment:
can you please add an explanation why this is called the same name as the one in `gc_heap`?

Reply:
I added a comment to that effect as well.